assets/zstandard/main.py

```
@@ -0,0 +1,111 @@
+        ingestout["decompress_failed"] = "yes" if failed else "no"
+        ingestout["decompress_start_time"] = datetime.datetime.fromtimestamp(self.bench_info['start_time']).strftime('%Y-%m-%d %H:%M:%S')
+
+        self.output.write()
+
+
+    def clear_cache(self):
+        """
+        Clears the cache within the docker container for cold run
+        """
+        self.docker_execute("sync")
+        self.docker_execute("echo 1 >/proc/sys/vm/drop_caches", check=False, shell=True)
```

Ruff (0.12.2)

Function call with `shell=True` parameter identified, security issue

(S604)